# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/svg/crashtests/1601824.html
# Commit: a061c1e11b76
# Full Hash: a061c1e11b7643628db9da237a10c0251e177f99
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-12-12 09:53:26
# Description:
#   Bug 1601824 - Measure partial ligature advance contributions correctly on text paths. r=longsonr
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56665
# ==============================================================================

diff -r 749aeb731dd9 -r a061c1e11b76 layout/svg/crashtests/1601824.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/svg/crashtests/1601824.html	Wed Dec 11 04:57:41 2019 +0000
@@ -0,0 +1,7 @@
+<svg>
+<polygon id="a" points="0 1">
+</polygon>
+<text>
+<textPath xlink:href="#a">
+<marker />
+<a systemLanguage="">